File: backend/Entities/Models/TransportadorLastMile.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class TransportadorLastMile : BaseEntity
{
    public Guid Id { get; set; }

    public int TransportadorId { get; set; }

    public string? CepInicial { get; set; }

    public string? CepFinal { get; set; }

    public int? BairroId { get; set; }

    public int? MunicipioId { get; set; }

    public int? EstadoId { get; set; }

    public DateTime? DataCadastro { get; set; }

    public virtual Estado? Estado { get; set; }

    public virtual ICollection<TransportadorLastMileGrupoItem> TransportadorLastMileGrupoItems { get; } = new List<TransportadorLastMileGrupoItem>();
}

File: frontend-with-menus/src/app/Business/Models/transportadorLastMile.ts
import { Estado } from "./estado";
import { TransportadorLastMileGrupoItem } from "./transportadorLastMileGrupoItem";

export interface TransportadorLastMile  {
    id: string;
    transportadorId: number;
    cepInicial: string | null;
    cepFinal: string | null;
    bairroId: number | null;
    municipioId: number | null;
    estadoId: number | null;
    dataCadastro: string | null;
    estado: Estado | null;
    transportadorLastMileGrupoItems: TransportadorLastMileGrupoItem[];
}